bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx96-de.html

```diff
@@ -0,0 +1,135 @@
+          <img class="mzp-c-picto-image" src="{{ static('img/firefox/whatsnew/whatsnew-96-eu/monitor.svg') }}" loading="lazy" alt="">
+          <h2 class="mzp-c-picto-heading">Firefox Monitor benachrichtigt dich bei Datenlecks</h2>
+          <div class="mzp-c-picto-body">
+            <p>Überprüfe, ob du schon mal von einem bekannten Datenleck betroffen warst und lass dich zukünftig warnen, wenn deine Daten ein Teil eines Datenlecks werden.</p>
+          </div>
+        </div>
+
+        <div class="mzp-c-picto">
+          <img class="mzp-c-picto-image" src="{{ static('img/firefox/whatsnew/whatsnew-96-eu/sync.svg') }}" loading="lazy" alt="">
+          <h2 class="mzp-c-picto-heading">Nahtloses Surfen auf allen Geräten mit Sync</h2>
+          <div class="mzp-c-picto-body">
+            <p>Greife auf deine Lesezeichen, Passwörter und offenen Tabs zu – egal wo du bist und bei gewohnt starkem Firefox Datenschutz.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="show-fxa-supported-signed-in">
+      <div class="mzp-l-content mzp-t-content-lg mzp-l-columns mzp-t-columns-three">
+        <div class="mzp-c-picto">
+          <img class="mzp-c-picto-image" src="{{ static('img/firefox/whatsnew/whatsnew-96-eu/de/relay-extension.svg') }}" loading="lazy" alt="">
+          <h2 class="mzp-c-picto-heading">Nutze die Erweiterung</h2>
+          <div class="mzp-c-picto-body">
+            <p>Hol dir die Relay-Erweiterung für den Firefox Browser und erstelle automatisch Aliasse, wann immer eine Webseite nach deiner E-Mail-Adresse fragt. Mit nur einem Klick.</p>
+          </div>
+        </div>
+
+        <div class="mzp-c-picto">
+          <img class="mzp-c-picto-image" src="{{ static('img/firefox/whatsnew/whatsnew-96-eu/de/relay-email-alias.svg') }}" loading="lazy" alt="">
+          <h2 class="mzp-c-picto-heading">Erstelle ein neues Alias</h2>
+          <div class="mzp-c-picto-body">
+            <p>Nutze die E-Mail-Aliasse, wo immer du deine private E-Mail-Adresse ungern angeben möchtest. Z. B. bei unbekannten Online Shops oder einem Newsletter, den du testen möchtest.</p>
+          </div>
+        </div>
+
+        <div class="mzp-c-picto">
+          <img class="mzp-c-picto-image" src="{{ static('img/firefox/whatsnew/whatsnew-96-eu/de/relay-manage-address.svg') }}" loading="lazy" alt="">
+          <h2 class="mzp-c-picto-heading">Verwalte deine Aliasse</h2>
+          <div class="mzp-c-picto-body">
+            <p>Bis zu 5 Wegwerfadressen leiten Nachrichten zuverlässig an deine echte E-Mail weiter, bis du genug hast und die Aliasse deaktivierst oder löschst.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="mzp-c-picto wnp-lead-out">
+    <img class="mzp-c-picto-image" src="{{ static('img/firefox/whatsnew/whatsnew-96-eu/8-bit-heart.svg') }}" loading="lazy" alt="">
+    <h3 class="mzp-c-picto-heading">Aus Liebe zum Web und aus Respekt vor dir.</h3>
+  </div>
+
+</section>
```

Comment:
**nit:** This floating closing tag isn't related to an opening tag -- let's remove?